pinc/TallyBoard.inc

```diff
@@ -21,9 +21,19 @@
 
 // If a holder isn't explicitly recorded in a TallyBoard,
 // then it is implicitly recorded with a tally of zero.
-// So  the TallyBoard is prepared to answer questions about
+// So the TallyBoard is prepared to answer questions about
 // holders it has never heard of before.
 
+// TallyBoards store and access data in 5 tables, 4 of which treat the
+// holder_type/holder_id/tally_name tuple as a primary key:
+// * current_tallies - stores the current tally, one per tuple
+// * past_tallies - stores historical tallies of deltas, one row per tuple
```

Comment:
The primary key of `past_tallies` isn't `holder_type/holder_id/tally_name`, it also includes `timestamp`.